cli/Cargo.toml

```diff
@@ -3,6 +3,18 @@ name = "didkit-cli"
 version = "0.1.0"
 authors = ["Spruce Systems, Inc."]
 edition = "2018"
+description = "Command-line interface for Verifiable Credentials and Decentralized Identifiers."
+license = "Apache-2.0"
+license-file = "../LICENSE"
```

Comment:
Does this work, or do we need to symlink inside again?

Reply:
It appears to work, as the publish dry-run results in the tarball containing the license file.